src/components/ex-link-list/ex-link-list.scss

```diff
@@ -0,0 +1,53 @@
+.linklist {
+  @apply w-full p-8 my-8;
```

Comment:
The margin-y of the linklist differs from the download list. I would use the same margin values as the download component. Moreover, I'm not sure about the margin-bottom in the linklist and download component. Hasn't it become standard to have no margin-bottom, just margin-top?

Reply:
Changed it to mt-component-spacing-small. Also created a task https://jira.oegbverlag.at/browse/EX-1595 because the component spacing is not the same for all our components. My bad